### ISSUE TITLE
bug: Google Map Marker unresponsive to touch

### ISSUE THREAD

**Issue by ray023** [driftyco/ionic#3391]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>ios</span> <span ionic-platform-version>8</span>  <span ionic-webview>webview</span>

<span ionic-description>Google Map Markers go unresponsive to touch on iPhone5 when referencing the ionic bundle.  

_NOTE:_  It still works on PC when using command <i>ionic serve</i>

[All inclusive example](https://www.dropbox.com/s/4dg8l9ca0uq83r9/Google%20Marker%20has%20unresponsive%20tap%20with%20Ionic.zip?dl=0)

[The CodePen version](http://codepen.io/anon/pen/VYgpwg)  but bear in mind, this will work.  It's only unresponsive on mobile device.

In the example's default state (e.g. referencing AngularJS, not Ionic), you can test on mobile device and touching the marker will bring up the "title".

After verifying that works, then make the following changes to reference ionic:
1.  From this:
```
<!--********************************************
     Toggle to reference just angular (no ionic)
*******************************************-->
<script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.3.14/angular.min.js"></script>

<!--***************************************
Toggle to reference ionic
***************************************-->
<!--<script src="http://code.ionicframework.com/1.0.0-rc.1/js/ionic.bundle.min.js"></script>-->
```
To this:
```
<!--********************************************
     Toggle to reference just angular (no ionic)
*******************************************-->
<!--<script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.3.14/angular.min.js"></script>-->

<!--***************************************
Toggle to reference ionic
***************************************-->
<script src="http://code.ionicframework.com/1.0.0-rc.1/js/ionic.bundle.min.js"></script>
```

And this:
```
    //IONIC NOT REFERENCED
    angular.module('app', ['uiGmapgoogle-maps'])
    //IONIC REFERENCED
    //angular.module('app', ['ionic','uiGmapgoogle-maps'])
```
To this:
```
    //IONIC NOT REFERENCED
    //angular.module('app', ['uiGmapgoogle-maps'])
    //IONIC REFERENCED
    angular.module('app', ['ionic','uiGmapgoogle-maps'])
```

Now run.  The map will still load, the markers will still show.  However, the labels do not appear when you touch them.</span>

<span is-issue-template></span>

**Comment by Ionitron** [comment]
Greetings @ray023!

My sensors indicate that you need to update your issue through our [custom issue form](http://ionicframework.com/submit-issue/#/?iid=3391). We are now requiring all issues to be submitted this way, to ensure that we have all of the information necessary to fix them as quickly as possible.

**[Click Here To Update Your Issue](http://ionicframework.com/submit-issue/#/?iid=3391)**

I will have no choice but to close this issue if it is not resubmitted through the form. Please fill out the rest of the form, so that I may use my friendly robot powers to assist you.

Thank you!

<span ionitron-issue-resubmit></span>

**Comment by Fayozjon** [comment]
Ray

http://wenxuebu.com/option_dragon

24.03.2015, 17:47, "Ray Nowell" <notifications@github.com>:Google Map Markers go unresponsive to touch on my iPhone5 when referencing the ionic bundle.  NOTE:  It still works on PC when using command ionic serve



In the example's default state (e.g. referencing AngularJS, not Ionic), you can test on mobile device and touching the marker will bring up the "title".

After verifying that works, then make the following changes to reference ionic:
1.  From this:


<script src=";</script>





To this:





<script src=";</script>


And this:

    //IONIC NOT REFERENCED
    angular.module('app', ['uiGmapgoogle-maps'])
    //IONIC REFERENCED
    //angular.module('app', ['ionic','uiGmapgoogle-maps'])


To this:

    //IONIC NOT REFERENCED
    //angular.module('app', ['uiGmapgoogle-maps'])
    //IONIC REFERENCED
    angular.module('app', ['ionic','uiGmapgoogle-maps'])


Now run.  The map will still load, the markers will still show.  However, the labels do not appear when you touch them.

**Comment by perrygovier** [comment]
Does adding [`data-tap-disabled="true"`](http://ionicframework.com/docs/api/page/tap/) to `#map_canvas` help?

**Comment by ray023** [comment]
Why yes; yes it does.  

Works like a charm.  Tyvm!
